pix-editor/app/controllers/authenticated/missions/new.js

```diff
@@ -0,0 +1,25 @@
+import Controller from '@ember/controller';
+import { action } from '@ember/object';
+import { inject as service } from '@ember/service';
+
+export default class MissionController extends Controller {
+  @service router;
+  @service notifications;
+
+  @action
+  async createMission(formData) {
+    try {
+      await this.model.mission.save({ adapterOptions: formData });
```

Comment:
**suggestion:** Pour être plus proche du *ember way*, il faudrait assigner les données de `formData` dans `this.model.mission` et ensuite appeler le `save` sans mettre les données dans le `adapterOptions`.

Reply:
Nous avons fait comme dans `static-courses`. Nous venons d'essayer comme tu nous le suggère, mais nous n'avons pas réussi à faire fonctionner le code correctement. 

À l'occasion, si tu as le temps, tu pourras nous montrer la voie d'ember ?